### DISCUSSION TITLE
Automation: create row with fields reffering differents tables

### DISCUSSION THREAD

**Discussion by aannddrreeaa** [Budibase/budibase#8349] [Help]
Hi, I'm Andrea. I'm pretty new in Budibase. I'm trying to build an automation when a row is created.
Currently, my DB in composed by a big table of input data ("Input" table) full of computers information and a couple of other tables (table "computer" and table "type of computer") related to each other by a field. (types of computer table can have many computer table rows, and a computer can only refer to a computer type)
Manually, adding in a row (in the "computer" table) his data and his own category, budibase allows me to select the type of computer from the "type of computer" table.
But now I need this to work in an automation.
So, my automation works like: when a row in "Input" table is created, create a new row in  "Computer" table with only a selection of values.
I use Bindings to get values and I tested that when I fill a simple field as "Serial number" it works correctly, I just take a value of a field from the new "Input" table row and return it. 
But when I fill a field  linked to foreign key in  "Type of computer" table, nothing happens (not even an error  message)

Is it possible to do that kind of thing in javascript?

![pics](https://user-images.githubusercontent.com/93607238/197179594-cd3f8807-5256-45e2-bd4b-b9aae7030ee2.png)

**Comment by melohagan** [comment]
Hey @aannddrreeaa 

Thanks for giving Budibase a go!

I assume you are using a SQL database such as Postgres? Would you be able to share screenshots of your table schemas as well as the JavaScript functions used in your bindings please?
